### PR TITLE
added examples for some of the run commands

### PR DESCRIPTION
**Pull request by SamMarkowitz**
Fixes CloudSlang/cloud-slang#225

Signed-off-by: Sam Markowitz <sam.markowitz@hp.com>

### CODE REVIEW

File: app/markdowns/slang.md [hunk 1]
```diff
@@ -1411,6 +1411,10 @@ cslang>run --f c:/.../your_flow.sl --i input1=root,input2=25
 ```
 Commas (`,`) can be used as part of input values by escaping them with a backslash (`\`).
 
+```bash
+cslang>run --f c:/.../your_flow.sl --i list=1,2,3
+```
```
Comment (orius123) [inline app/markdowns/slang.md]: You forgot to escape

File: app/markdowns/slang.md [hunk 2]
```diff
@@ -1418,6 +1422,10 @@ cslang>run --f c:/.../your_flow.sl --i "input1=Hello World, input2=x"
 
 Double quotes (`"`) can be used as part of quoted input values by escaping them with a backslash (`\`). When using a quoted input list, spaces between input parameters will be trimmed.
 
+```bash
+cslang>run --f c:/.../your_flow.sl --i "input1=\"Hello\" World"
```
Comment (orius123) [inline app/markdowns/slang.md]: Maybe worth showing what it is actually printing

File: app/markdowns/slang.md (hunk 1 above)
Comment (SamMarkowitz) [inline app/markdowns/slang.md]: Oops. That's embarrassing.

**Comment by SamMarkowitz** [comment]
@orius123 - does this do the trick?

**Pull request merged**